MyAddressBook-final/app/src/main/java/com/example/android/myaddressbook/ContactsActivity.kt

```diff
@@ -59,6 +57,9 @@ class ContactsActivity : AppCompatActivity(), TextWatcher {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_contacts)
 
+        val toolbar = findViewById<Toolbar>(R.id.toolbar)
```

Comment:
nit: you only use those once, later on
maybe there's no need for saving them to a variable? 
or in any case, maybe move the declaration closer to the actual usage?

Reply:
keeping this as it is, so we don't need to change all the steps and the codelab text just for this.